### ISSUE TITLE
multiple polar scans as input

### ISSUE THREAD

**Issue by adokter** [adokter/vol2bird#116]
* [x] Update parsing of command line options. Switch to using `getopt_long()` from `<getopt.h>`
* [x] Detect for file existence
* [x] Write function that detects whether filetype is ODIM, RSL or IRIS. See `isIRIS` in iris2odim library.
* [x] Write function that detects whether file is a scan or a volume
* [x] Write `PolarVolume_t* vol2birdGetRSLVolume()` for librsl.c
* [x] Write `PolarVolume_t* vol2birdGetIRISVolume()` for libvol2bird.c
* [x] Write `PolarVolume_t* vol2birdGetODIMVolume()` for libvol2bird.c

**Comment by adokter** [comment]
`access()` function, found in unistd.h:

```
if( access( fname, F_OK ) != -1 ) {
    // file exists
} else {
    // file doesn't exist
}
```

**Comment by adokter** [comment]
closed by commit c1d3fb5fb95409b9af5a6bf09a541abe18991958

**Comment by adokter** [comment]
This partly addresses issue #84
